0008-interledger-quoting-protocol/0008-interledger-quoting-protocol.md

```diff
@@ -33,108 +31,90 @@ The sending and receiving ledger are different ledgers. Otherwise, there should
-| `data`   | QuoteParameters | An Object describing the quote parameters. |
+## ILQP Packets
+### IlqpLiquidityRequest
 
-##### QuoteParameters
+| Field | Type | Short Description |
+|:--|:--|:--|
+| destinationAccount | Address | Address corresponding to the destination account |
+| destinationHoldDuration | UInt32 | How much time the receiver needs to fulfill the payment (in milliseconds) |
 
-| Field                         | Type        | Description |
-|:------------------------------|:------------|:------------|
-| `source_address`              | ILP Address | Address of the source's account in the source ledger. |
-| `destination_address`         | ILP Address | Address of the destination's account in the destination ledger. |
-| `source_amount`               | String      | Fixed integer amount to debit from the source's account. (Required unless `destination_amount` specified.) |
-| `destination_amount`          | String      | Fixed integer amount to credit to the destination's account. (Required unless `source_amount` specified.) |
-| `destination_expiry_duration` | String      | (Optional) Number of milliseconds before the transfer in the destination ledger expires. |
-| `source_expiry_duration`      | String      | (Optional) Number of milliseconds before the transfer in the source ledger expires. |
-| `slippage`                    | String      | (Optional) Custom slippage to assume for the exchange between ledgers, as a decimal proportion. Lower slippage means the payment is cheaper but more likely to fail due to market movement. |
+### IlqpLiquidityResponse
 
-The request must specify either `source_amount` or `destination_amount` but not both.
+| Field | Type | Short Description |
+|:--|:--|:--|
+| liquidity | LiquidityCurve | Curve describing the liquidity for the quoted route |
+| appliesToPrefix | Address | Common prefix of all addresses for which this liquidity curve applies. |
+| sourceHoldDuration | UInt32 | How long the sender should put the money on hold (in milliseconds) |
+| expiresAt | Timestamp | Maximum time where the connector expects to be able to honor this liquidity curve |
 
-##### Example request
+`LiquidityCurve` is a `SEQUENCE OF SEQUENCE { x UInt64, y UInt64 }`
 
-```js
-{
-  "method": "quote_request",
-  "id": "721e4126-98a1-4974-b35a-8a8f4655f934",
-  "data": {
-    "source_amount": "10025",
-    "source_address": "example.eur-ledger.alice",
-    "destination_address": "example.usd-ledger.bob",
-    "source_expiry_duration": "6000",
-    "destination_expiry_duration": "5"
-  }
-}
-```
+### IlqpBySourceRequest
+
+| Field | Type | Short Description |
+|:--|:--|:--|
+| destinationAccount | Address | Address corresponding to the destination account |
+| sourceAmount | UInt64 | Amount the sender needs to send, denominated in the asset of the source ledger |
+| destinationHoldDuration | UInt32 | How much time the receiver needs to fulfill the payment (in milliseconds) |
```

Comment:
so amount now goes source->dest, but hold duration still goes dest->source? why is this?